### PR TITLE
Fix apis

### PR DESCRIPTION
**Pull request by dairyo**
* Fixed query APIs.
* Fixed API documents.

Related PR: #265

### CODE REVIEW

File: ThingIFSDK/ThingIFSDK/AggregatedResult.swift [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 import Foundation
 
+/** Aggregated result. */
 open class AggregatedResult<AggregatedValueValue>: NSCoding {
```
Comment (yongpingchen) [inline ThingIFSDK/ThingIFSDK/AggregatedResult.swift]: Name of `AggregatedValueValue`  seems strange. Please fix the name

Comment (dairyo) [reply ThingIFSDK/ThingIFSDK/AggregatedResult.swift]: Thanks.

**Comment by dairyo** [comment]
@yongpingchen 
fixed according to your comment. Please check it.

**Comment by yongpingchen** [comment]
LGTM

**Pull request merged**